src/components/LessonSlider.css

```diff
@@ -1,7 +1,14 @@
-.content {
+.slide-content {
```

Comment:
Give naming convention more precise so in future we won't get this kind of issues do same changes to master PR also

like : **lesson-slider-content**